File: API_SC_01_08Abril/API-Karoline-Raiane-da-Silva-Nascimento/q5_pesquisa.py
def main():

    menu()

    valor_total = 0
    lista = ""

    escolha = int(input("Escolha uma opção: "))

    while escolha == 1:
        descricao = input("Descrição do item: ")
        especificacao = input("Especificação do produto: ")
        valor = int(input("Valor do produto: "))
        
        n_valor = str(valor)
        valor_total += valor
        lista += f"{descricao} ({especificacao}) R$ {valor} \n"

        escolha = int(input("Adicionar mais um item? (1 - Sim; 0 - Não) "))

        if escolha == 0:
            print("Fim da lista")
        else:
            descricao = input("Descrição do item: ")
            especificacao = input("Especificação do produto: ")
            valor = int(input("Valor do produto: "))


    resultado = f"""
    ------------------------------------
    {lista}
    ------------------------------------
    Valor total: R$ {valor_total}
    Opções de pagamento: {calc_forma_pagamento(valor_total)}"""

    print(resultado)
    


def menu():
    menu = """
    1 - Incluir Item
    0 - Sair"""

    print(menu)

def calc_forma_pagamento(valor_total):
    parc_3 = valor_total / 3
    parc_6 = valor_total / 6

    if valor_total <= 30:
        return "A vista"
    elif valor_total <= 100:
        return f"Parcelamento em até 3x de R$ {parc_3:.2f}"
    elif valor_total > 100:
        parcelas = int(input("Em quantas parcelas deseja dividir? "))
        valor_juros = valor_total + (valor_total * 0.05 ** parcelas)
        valor_total = valor_juros
        return f"Parcelado em {parcelas} vezes com o total de {valor_total:.2f}"

main()

File: API_SC_01_08Abril/Isaac Borges/q5_pesquisa.js
import { get_entry, get_number } from "./api_utils.js"

function get_menu(){

    const escolha = get_number('O que vai fazer?  : ')

    console.log(menu)
    return escolha
}


function main(){
    let precototal = 0
    let valor = 0
    let descricao = ''
    let especificacao = ''
    let lista = `-------------------------------------`
    let escolha = get_menu()

    if (escolha === 1) {
        lista
        descricao = get_entry('Descrição do item: ')
        especificacao = get_entry('Especificação do item: ')
        valor = get_number('Valor do item: ')
        precototal += valor
        
        lista += `
        ${descricao} (${especificacao})` + `R$${valor.toFixed(2)}`

        escolha = get_menu()
    } if (escolha === 2) {
        `
        --------------------------------------------
        Valor total:                        R$${precototal}`
    }
}

main()